### PR TITLE
Add some tests for formIndex route and remove HAR files

### PR DESCRIPTION
**Pull request by PascalinDe**
This adds some initial tests for formIndex route and more importantly removes the HAR files in the mocks to rely on simplified JSON files instead

### CODE REVIEW

File: web/frontend/tests/mocks.ts [hunk 1]
```diff
@@ -50,3 +51,29 @@ export async function mockProxy(page: any) {
     });
   });
 }
+
+export async function mockEvoting(page: any, empty: boolean = true) {
```
[the remark this answers is not in the document]
Comment (PascalinDe) [reply web/frontend/tests/mocks.ts:54]: OK, I think I misunderstood Typescript, I thought there'd be only type annotations possible if there is a `@types` package?

[the remark this answers is not in the document]
Comment (ineiti) [reply web/frontend/tests/mocks.ts:54]: > @types
> https://github.com/DefinitelyTyped/DefinitelyTyped​ is definitely one of TypeScript's greatest strengths. The community has effectively gone ahead and documented the nature of nearly 90% of the top JavaScript projects out there.

So it's only needed for existing javascript projects which don't have an official types entry. For typescript projects, like `playwright`, it's not needed.

[the remark this answers is not in the document]
Comment (PascalinDe) [reply web/frontend/tests/mocks.ts]: thanks for the link! I updated the types

**Review by ineiti — APPROVED** [review]
So beautiful - no `any`!

**Pull request merged**